app/models/user.rb

```diff
@@ -22,12 +22,17 @@ class User < ApplicationRecord
   # TODO Which fields do we want updated each time a user logs in?
   #   - Group memberships
   def self.from_omniauth(auth)
-    User.find_or_create_by(provider: auth.provider, uid: auth.uid) do |new_user|
+    user = User.find_or_create_by(provider: auth.provider, uid: auth.uid) do |new_user|
       new_user.access_id = auth.info.access_id
       new_user.email = auth.info.email
+      new_user.ldap_groups = auth.info.groups
       new_user.given_name = auth.info.given_name
       new_user.surname = auth.info.surname
     end
+
+    user.update(ldap_groups: auth.info.groups)
```

Comment:
I'm curious if we maybe only want to run this operation if authproxy returns groups && they differ from what's already in the database? @rschenk Thoughts on this one?